### DISCUSSION TITLE
Get the Community to help tagging issues

### DISCUSSION THREAD

**Discussion by kmf** [pop-os/cosmic-epoch#1111] [Ideas]
Hi there,

There is a lot of issues relating to different sub projects, in this repo, perhaps we can help by tagging issues based on the repo they belong to eg. `cosmic-files` or `cosmic-settings`

Maybe a step further could be to tag things based on `Bug` or `Feature`

kmf

**Comment by leviport** [comment]
Tagging bugs/features and transferring to different repos is easy enough. The thing that takes the most bandwidth is handling duplicate bug reports and feature requests, since we have to hunt down the original issues to link when closing the duplicates. Community members have been pretty helpful with this, however. All they need to do is mention that an issue is a duplicate and link to the original, and one of us can close it pretty quickly without having to search around for the original issue.
